bemuse/src/app/ui/OptionsAdvanced.tsx

```diff
@@ -0,0 +1,75 @@
+import './OptionsAdvanced.scss'
+
+import * as Options from '../entities/Options'
+
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import OptionsButton from './OptionsButton'
+import OptionsInputField from './OptionsInputField'
+import { selectOptions } from '../redux/ReduxState'
+
+const stringifyLatency = (latency: number): string => {
+  return Math.round(latency) + 'ms'
+}
+
+const parseLatency = (latencyText: string): number => {
+  return parseInt(latencyText, 10)
+}
+
+const useLatency = (handler: (latency: number) => void) => {
+  useEffect(() => {
+    const handleMessage = (event: MessageEvent) => {
+      if (event.data && typeof event.data.latency === 'number') {
+        handler(event.data.latency)
+      }
+    }
+    window.addEventListener('message', handleMessage)
+    return () => {
+      window.removeEventListener('message', handleMessage)
+    }
+  }, [])
```

Comment:
Strange… ESLint should have flagged that `handler` should go into the dependency array. Maybe I forgot to enable the new React rules in ESLint. Anyways, I will merge this PR as-is because the current code isn’t buggy. Will add the ESLint rules later 😃